fix(bil): docs update (#4485)
scaleway/docs-content: pages/billing/troubleshooting/fix-common-billing-issues.mdx

---
meta:
  title: Fix common billing issues

>   description: Resolve issues with your invoices

Reviewer: ```suggestion
  description: Troubleshoot issues preventing you from paying your Scaleway invoice.
```

content:
  h1: Fix common billing issues

>   paragraph: Resolve issues with your invoices

Reviewer: ```suggestion
  paragraph: Troubleshoot issues preventing you from paying your Scaleway invoice.
```

tags: billing credit card invoice
dates:
  validation: 2025-02-24
  posted: 2025-02-24
categories:
  - billing
---

## I cannot pay my invoice

### Cause
Your payment may have failed due to insufficient funds, an issue with your payment method, or incorrect SEPA mandate details.

### Solution
**If you are paying by credit card, try the following:**
 - Ensure you have enough funds in your bank account. 
 - If payment still does not work, the reason for the failure is indicated in the error message in the console
<Message type="tip">

>   A "card_declined" error message means the payment was refused by your bank. Contact your bank for more details.

Reviewer 2: ```suggestion
  A `card_declined` error message means the payment was refused by your bank. Contact your bank for more details.
```

</Message>

**If you are paying via SEPA direct debit, try the following:**
- Verify that your SEPA mandate is correctly set up to ensure future payments are processed.
- Verify that your billing address matches the country of your bank account.
- Retry the payment.

> <Message type="requirement">

Reviewer: ```suggestion
<Message type="important">
```

  Any unpaid invoice must be paid by credit card.
</Message>

> ## I didn't receive my invoice

Reviewer: ```suggestion
## I did not receive my invoice
```

>   Any unpaid invoice must be paid by credit card.

Reviewer 2: ```suggestion
  Unpaid invoices must be paid by credit card.
```

### Cause
Your invoice may not have been delivered due to an incorrect billing contact email or an issue with email delivery.

### Solution
**Verify your billing email address:**
- Check the [billing contact](/faq/billing/#how-can-i-add-a-billing-contact-to-my-account) section in your Scaleway console to ensure the email address is correct. 
- Check your spam or junk folder.
<Message type="tip">
  You can also [download the your invoices](/billing/how-to/download-an-invoice/) from the console.
</Message>

## I have no access to my invoices 

### Cause
Access to invoices is restricted based on your role and permissions within the Scaleway Organization. Only users with the necessary IAM rights can view the billing section.

### Solution

> - [Check your permissions within your Organization](/iam/quickstart/#how-to-give-permissions-to-users-and-applications-via-policies): if you are not the owner of the Organization, ensure that you have the appropriate IAM rights to access the Billing section.

Reviewer: ```suggestion
- [Check your permissions within your Organization](/iam/quickstart/#how-to-give-permissions-to-users-and-applications-via-policies): if you are not the owner of the Organization, ensure that you have the appropriate IAM rights to access the billing section.
```
